javav2/example_code/cloudwatch/src/main/java/com/example/cloudwatch/CloudWatchScenario.java

```diff
@@ -0,0 +1,982 @@
+        System.out.println("8. Add an additional metric to the dashboard.");
+        addMetricToDashboard(cw, dashboardAdd, dashboardName);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("9. Create an alarm for the custom metric.");
+        String alarmName = createAlarm(cw, settings);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("10. Describe 10 current alarms.");
+        describeAlarms(cw);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("11. Get current data for new custom metric.");
+        getCustomMetricData(cw,settings);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("12. Push data into the custom metric to trigger the alarm.");
+        addMetricDataForAlarm(cw, settings) ;
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("13. Check the alarm state using the action DescribeAlarmsForMetric.");
+        checkForMetricAlarm(cw, settings);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("14. Get alarm history for the new alarm.");
+        getAlarmHistory(cw, settings, myDate);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("15. Add an anomaly detector for the custom metric.");
+        addAnomalyDetector(cw, settings);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("16. Describe current anomaly detectors.");
+        describeAnomalyDetectors(cw, settings);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("17. Get a metric image for the custom metric.");
+        getAndOpenMetricImage(cw, metricImage);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("18. Clean up the Amazon CloudWatch resources.");
+        deleteDashboard(cw, dashboardName);
+        deleteCWAlarm(cw, alarmName);
+        deleteAnomalyDetector(cw, settings);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("The Amazon CloudWatch example scenario is complete.");
+        System.out.println(DASHES);
+        cw.close();
+    }
+
+    // snippet-start:[cloudwatch.java2.scenario.del.anomalydetector.main]
+    public static void deleteAnomalyDetector(CloudWatchClient cw, String fileName) {
+        try {
+            // Read values from the JSON file.
+            JsonParser parser = new JsonFactory().createParser(new File(fileName));
+            com.fasterxml.jackson.databind.JsonNode rootNode = new ObjectMapper().readTree(parser);
+            String customMetricNamespace = rootNode.findValue("customMetricNamespace").toString().replaceAll("\"", "");

```

Comment:
As discussed in chat, you can use .asText and not have to do the replaceAll every time from the json values.